devops/jobs/CreateSandboxCI.groovy

```diff
@@ -61,8 +61,6 @@ class CreateSandboxCI {
                   booleanParam('reconfigure',true)
                   booleanParam('recreate',false)
                   predefinedProp('name_tag','continuous-integration')
-                  booleanParam('xqueue',true)
-                  booleanParam('xserver',true)
```

Comment:
Right, I keep forgetting that.  I should really have made it a standalone task that just runs a subset of ansible-provision.sh.  Some other day.